client/src/views/home/HomePage.vue

```diff
@@ -256,7 +259,26 @@ async function openCreateOrJoin(event: Event): Promise<void> {
 
 async function refreshDeviceList(): Promise<void> {
   querying.value = true;
-  deviceList.value = await listAvailableDevices();
+  const result = await listAvailableDevicesWithError();
+  if (!result.ok) {
+    let message = 'HomePage.organizationList.errors.generic';
+    if (result.error.tag === ListAvailableDeviceErrorTag.StorageNotAvailable) {
+      if (isWeb()) {
+        message = 'HomePage.organizationList.errors.noStorageWeb';
+      } else {
+        message = 'HomePage.organizationList.errors.noStorageDesktop';
+      }
+    }
```

Comment:
We can have the same error message between web and desktop.
```
"Unable to access your organizations: the application storage is not available.",
```

And add a "detail" string to list the things the user can check to avoid the issue.